api/v1/tetragon/tetragon.proto

```diff
@@ -321,6 +321,10 @@ message KprobeCred {
     repeated CapabilitiesType inheritable = 3;
 }
 
+message KprobeLinuxBinprm {
+    string path = 1;
```

Comment:
should these be enum or const values vs literals?

Reply:
Probably yes, and this problem exists in other `.proto` files as well, but in this PR, I'd like to be as consistent with their existing style as possible. If we want to